shared_models/shared_models.py

```diff
@@ -287,9 +287,7 @@ def v1_state_to_rfc_state(state: Optional[str]) -> Optional[str]:
         "request_received": "request-received",
         "credential_issued": "credential-issued",
         "credential_received": "credential-received",
-        # V1 uses credential_acked, while v2 uses done
-        "credential_acked": "done",
-        "credential_revoked": "credential-revoked",
+        "credential_acked": "credential-acked",
```

Comment:
I think we should map the states to the same values for both v1 and v2. Was this causing issues for the tests? Now the done state for v1 is `credential-acked` while for v2 it's `done`.